src/noise_fns/generators/fractals/basicmulti.rs

```diff
@@ -34,8 +33,8 @@ pub const BASICMULTI_MAX_OCTAVES: usize = 32;
 /// smooth. As the value moves further away from zero, higher frequencies will
 /// not be as damped and thus will grow more jagged as iteration progresses.
 ///
-#[derive(Clone, Debug)]
-pub struct BasicMulti<T> {
+#[derive(Clone, Debug, Default)]
```

Comment:
Deriving `Default` on this instead of using a manual implementation results in a useless struct. For example, using the default of `0.0` for `lacunarity` produces a result that has only 1 octave of noise regardless of the actual number of octaves, since any octaves other than the first always takes their noise sample at [0.0, N].